docs/content/docs/development/guide.md

```diff
@@ -93,6 +93,22 @@ rest.port: 8081
 rest.address: localhost
 ```
 
+### Uninstalling the operator locally
+```bash
+helm uninstall flink-operator
+```
+
+### CRD Generating and Upgrading 
+
+By default, the CRD is generated by the [Fabric8 CRDGenerator](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CRD-generator.md), when building from source.
+When installing flink-operator for the first time, the CRD will apply to the kubernetes cluster automatically. But it will not be removed or upgraded when re-install the flink-operator, as described in the relevant helm [documentation](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/). 
```

Comment:
Should be: 
the CRD will **be applied** to the kubernetes
upgraded when **re-installing** the flink-operator

Reply:
Fixed